6/ansible-6.0.0a3.deps

```diff
@@ -0,0 +1,101 @@
+_ansible_version: 6.0.0a3
+_ansible_core_version: 2.13.0
+amazon.aws: 3.2.0
+ansible.netcommon: 3.0.0
+ansible.posix: 1.3.0
+ansible.utils: 2.6.1
+ansible.windows: 1.10.0
+arista.eos: 5.0.0
+awx.awx: 21.0.0
+azure.azcollection: 1.12.0
+check_point.mgmt: 2.3.0
+chocolatey.chocolatey: 1.2.0
+cisco.aci: 2.2.0
+cisco.asa: 3.0.0
+cisco.intersight: 1.0.18
+cisco.ios: 3.0.0
+cisco.iosxr: 3.0.0
+cisco.ise: 2.4.0
+cisco.meraki: 2.6.2
+cisco.mso: 2.0.0
+cisco.nso: 1.0.3
+cisco.nxos: 3.0.0
+cisco.ucs: 1.8.0
+cloud.common: 2.1.1
+cloudscale_ch.cloud: 2.2.1
+community.aws: 3.2.1
+community.azure: 1.1.0
+community.ciscosmb: 1.0.5
+community.crypto: 2.3.1
+community.digitalocean: 1.19.0
+community.dns: 2.1.1
+community.docker: 2.5.1
+community.fortios: 1.0.0
+community.general: 5.0.0
+community.google: 1.0.0
+community.grafana: 1.4.0
+community.hashi_vault: 2.5.0
+community.hrobot: 1.3.0
+community.libvirt: 1.1.0
+community.mongodb: 1.4.0
+community.mysql: 3.2.1
+community.network: 3.3.0
+community.okd: 2.2.0
+community.postgresql: 2.1.5
+community.proxysql: 1.3.2
+community.rabbitmq: 1.2.1
+community.routeros: 2.0.0
+community.sap: 1.0.0
+community.skydive: 1.0.0
+community.sops: 1.2.1
+community.vmware: 2.4.0
+community.windows: 1.10.0
+community.zabbix: 1.6.0
+containers.podman: 1.9.3
+cyberark.conjur: 1.1.0
+cyberark.pas: 1.0.13
+dellemc.enterprise_sonic: 1.1.0
+dellemc.openmanage: 5.3.0
+dellemc.os10: 1.1.1
+dellemc.os6: 1.0.7
+dellemc.os9: 1.0.4
+f5networks.f5_modules: 1.16.0
+fortinet.fortimanager: 2.1.5
+fortinet.fortios: 2.1.5
```

Comment:
Isn't this the broken version?

Reply:
FTR, I'm talking about fortinet.fortios. I [asked the maintainers](https://github.com/fortinet-ansible-dev/ansible-galaxy-fortios-collection/commit/bbf08efdc24291665a1a506935565df53164d7dc#commitcomment-73041991) a little while ago to create a release with the fix, but they haven't done it.

Reply:
:crying_cat_face:

Reply:
:facepalm: you're right, it got bumped automatically, thanks for picking it up.

I'll manually pin it (again) and update the PR.

Reply:
Fixed in https://github.com/ansible-community/ansible-build-data/pull/123/commits/165555a57771f2e1808248bc590b1afb688bd3bc